ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/components/ExceptionRequestModal/utils.ts

```diff
@@ -83,3 +86,13 @@ export function formValuesToDeferralRequest(
             return ensureExhaustive(expiryType);
     }
 }
+
+export function formValuesToFalsePositiveRequest(
+    formValues: FalsePositiveValues
+): CreateFalsePositiveVulnerabilityExceptionRequest {
```

Comment:
I guess along the same lines as my previous comment, can we reuse types here since `FalsePositiveValues` and `CreateFalsePositiveVulnerabilityExceptionRequest` are the same?